scripts/simpledvt/02-clone-nor.js

```diff
@@ -0,0 +1,474 @@
+      .encodeABI(),
+  })
+
+  // grant manager to easytrack evm script executor
+  evmScriptCalls.push({
+    to: acl.address,
+    calldata: await acl.contract.methods
+      .createPermission(easyTrackEVMScriptExecutor, trgProxyAddress, MANAGE_SIGNING_KEYS, easyTrackEVMScriptExecutor)
+      .encodeABI(),
+  })
+
+  // grant perms to easytrack factories
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.AddNodeOperators,
+      trgProxyAddress +
+        getSignature(trgApp, 'addNodeOperator').substring(2) +
+        aclAddress.substring(2) +
+        getSignature(acl, 'grantPermissionP').substring(2),
+    ]),
+  })
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.ActivateNodeOperators,
+      trgProxyAddress +
+        getSignature(trgApp, 'activateNodeOperator').substring(2) +
+        aclAddress.substring(2) +
+        getSignature(acl, 'grantPermissionP').substring(2),
+    ]),
+  })
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.DeactivateNodeOperators,
+      trgProxyAddress +
+        getSignature(trgApp, 'deactivateNodeOperator').substring(2) +
+        aclAddress.substring(2) +
+        getSignature(acl, 'revokePermission').substring(2),
+    ]),
+  })
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.SetVettedValidatorsLimits,
+      trgProxyAddress + getSignature(trgApp, 'setNodeOperatorStakingLimit').substring(2),
+    ]),
+  })
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.UpdateTargetValidatorLimits,
+      trgProxyAddress + getSignature(trgApp, 'updateTargetValidatorsLimits').substring(2),
+    ]),
+  })
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.SetNodeOperatorNames,
+      trgProxyAddress + getSignature(trgApp, 'setNodeOperatorName').substring(2),
+    ]),
+  })
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.SetNodeOperatorRewardAddresses,
+      trgProxyAddress + getSignature(trgApp, 'setNodeOperatorRewardAddress').substring(2),
+    ]),
+  })
+  evmScriptCalls.push({
+    to: easytrack.address,
+    calldata: await easytrack.interface.encodeFunctionData('addEVMScriptFactory', [
+      easyTrackFactories.TransferNodeOperatorManager,
```

Comment:
In ET contracts this factory has `ChangeNodeOperatorManager` name, I would suggest sync the names